content/docs/command-reference/plots/modify.md

```diff
@@ -131,3 +135,25 @@ be changed in many cases. A simple command changes the template:
 ```dvc
 $ dvc plots modify classes.csv --template confusion
 ```
+
+## Example: Convert any output into a plot
+
+Let's take an example `evaluate` stage which has `logs.csv` as an output. We use
+`dvc plots modify` to convert the `logs.csv` output file into a plot, and then
+observe changes that happened in `dvc.yaml`:
```

Comment:
- We **can** use
- observe **the** changes

Reply:
I'm not sure "observe" is the right term though, how about "check" or "confirm"?

Reply:
"confirm" looks better